kedro-org/kedro-viz
# Feature Hint cards and app walkthrough

## Description

Resolves #1180.

## Development notes

I've added what I'm calling the `AppOnboarding` component and necessary files. I'm opening the PR a little early to get it into our hands so we can test it out. I feels suboptimal right now.

## QA notes

https://github.com/kedro-org/kedro-viz/assets/16869061/c29a48ab-2e2e-458a-990b-4fa21105188f

Look at the Gitpod link and play around with it and reference the design [here](https://www.figma.com/file/2WSNDAdG3JZS7qdaiH6Qz6/Kedro-Component-Library?type=design&node-id=102%3A8941&mode=dev).

I think there's a wrinkle or two with the concept itself. See my comment here for more details: https://github.com/kedro-org/kedro-viz/issues/1180#issuecomment-1638452433.

Other things that we can improve:
- The hit area of the small circles that are used to change to another card are much too small. It's too hard to click them.
- There's a lot of "to be decided" language in the design concept that hasn't been decided.
- The small "New feature" text element in the design doesn't make much sense to me, since a lot of what we want to show isn't actually a new feature. We can improve this language.

Still todo from the dev side:

- [ ] Write e2e and unit tests

## Checklist



<a href="https://gitpod.io/#https://github.com/kedro-org/kedro-viz/pull/1454"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
